test/acceptance/tests.py

```diff
@@ -762,25 +783,26 @@ def verify_submission_has_peer_grade(self, learner, max_attempts=5):
             max_attempts: the maximum number of times an additional peer grading should be done
         """
         count = 0
-        while self.grade_page.wait_for_page().score is None and count < (max_attempts + 1):
+        peer_page = AssessmentPage("peer-assessment", self.browser, self.problem_loc)
+        while not peer_page.is_complete and count < (max_attempts + 1):
             count += 1
             self.do_submission_training_self_assessment("extra_{}@looping.com".format(count), None)
             self.do_peer_assessment(options=self.PEER_ASSESSMENT)
             self.login_user(learner)
             self.grade_page.visit()
 
-        self.assertIsNotNone(
-            self.grade_page.wait_for_page().score,
+        self.assertTrue(
+            peer_page.is_complete,
```

Comment:
This allows the full workflow with staff required test to proceed properly after peer grading finishes.